src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -490,18 +602,12 @@ public void UpdateEnergyBalance(EnergyBalanceInfo energyBalance)
     public void UpdateOrganelleEfficiencies(
         System.Collections.Generic.Dictionary<string, OrganelleEfficiency> organelleEfficiency)
     {
-        foreach (var organelleName in organelleEfficiency.Keys)
+        foreach (var organelle in organelleEfficiency.Keys)
         {
-            foreach (Node tooltip in itemTooltipElements)
-            {
-                if (tooltip.Name == organelleName)
-                {
-                    var processList = tooltip.GetNode<VBoxContainer>("MarginContainer/VBoxContainer/ProcessList");
+            var tooltip = (SelectionMenuTooltip)tooltipHandler.GetTooltip(
+                SimulationParameters.Instance.GetOrganelleType(organelle).Name);
```

Comment:
Shouldn't the tooltip use InternalName otherwise things will explode if the name is tweaked?